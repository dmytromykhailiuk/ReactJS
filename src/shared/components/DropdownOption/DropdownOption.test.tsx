import { DropdownOption } from "./";
import { shallow } from "enzyme";
import React from "react";
import toJson from "enzyme-to-json";

describe("DropdownOption", () => {
  it("should match snapshot", () => {
    const wrapper = shallow(<DropdownOption />);
    expect(toJson(wrapper)).toMatchSnapshot()
  })

  it("should has dropdown-option__checkbox--checked class when isSelected", () => {
    const wrapper = shallow(<DropdownOption isSelected />);
    expect(wrapper.exists('.dropdown-option__checkbox--checked')).toBeTruthy()
  })
})
