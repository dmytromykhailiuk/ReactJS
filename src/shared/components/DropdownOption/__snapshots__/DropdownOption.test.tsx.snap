// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DropdownOption should match snapshot 1`] = `
<div
  className="dropdown-option"
>
  <div
    className="dropdown-option__checkbox"
  >
    <div
      className="dropdown-option__checkbox-icon"
    >
      <CheckboxIcon />
    </div>
  </div>
  <div
    className="dropdown-option__content"
  />
</div>
`;
